src/test/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMappingTest.java

```diff
@@ -76,6 +82,9 @@ class HearingsDetailsMappingTest extends HearingsMappingBase {
     private VenueService venueService;
 
     public static final String PROCESSING_VENUE_1 = "test_place";
+    private static final String PHONE_NUMBER = "07483871426";
+    private static final String HEARING_VENUE_ID_1 = "219164";
+    private static final String HEARING_VENUE_ID_2 = "436578";
```

Comment:
I would prefer HEARING_VENUE_ID_CURRENT and HEARING_VENUE_ID_OTHER. (if this makes sense for you here)
Generally speaking, it is good practice to avoid numbering variables, in favour of being descriptive as to what exactly they are representing. This is for when they are distinct. If you only need a bunch of generic variables, then numbering is fine. But in this case, because they both obviously serve a different purpose, I think it makes sense to label descriptively.

Reply:
Done